### PR TITLE
S390x 2.2: trivial fixes

### PR DESCRIPTION
**Pull request by clebergnu**
This is a collection of fixes and small improvements found while reviewing @ldoktor PR #1170 .

### CODE REVIEW

File: qemu/tests/multi_disk.py [hunk 1]
```diff
@@ -9,7 +9,6 @@
 import string
 
 from autotest.client.shared import error
-from autotest.client.shared import utils
```
Comment (ldoktor) [inline qemu/tests/multi_disk.py]: missing import astring

File: qemu/tests/multi_disk.py [hunk 2]
```diff
@@ -10,16 +10,17 @@
...
+from virttest import qemu_qtree
 from virttest import utils_misc
 
 _RE_RANGE1 = re.compile(r'range\([ ]*([-]?\d+|n).*\)')
 _RE_RANGE2 = re.compile(r',[ ]*([-]?\d+|n)')
 _RE_BLANKS = re.compile(r'^([ ]*)')
 
 
-@error.context_aware
+@error_context.context_aware
```
Comment (ldoktor) [inline qemu/tests/multi_disk.py]: ok

File: qemu/tests/cfg/qemu_img_negative.cfg [hunk 3]
```diff
@@ -20,8 +20,8 @@
                     host_setup_flag = 2
                     #Please update portal_ip, initiator and target params based on your test env
                     #portal_ip = 127.0.0.1
-                    #initiator = "iqn.2010-07.com.redhat:kvmautotest"
-                    #target = "iqn.2001-05.com.equallogic:0-8a0906-db31f7d03-470263b05654c204-kvm-autotest"
+                    #initiator = "iqn.2010-07.com.example:my_initiator"
+                    #target = "iqn.2001-05.com.mytarget:0-0a0000-000000000-0000000000000000-my-target"
```
Comment (ldoktor) [inline qemu/tests/cfg/qemu_img_negative.cfg]: Yep, good. Only please remove the `#` as without these the test simply uses the default local disk(s)...

File: qemu/tests/multi_disk.py (hunk 1 above)
Comment (clebergnu) [reply qemu/tests/multi_disk.py]: Yep, good catch.

**Pull request closed without merging**